### PR TITLE
Loss of keys - update of warnings to users

### PR DESCRIPTION
**Pull request by dockertopia**
Updating loss of keys warnings as per received request for change ENGDOCS-977.

### CODE REVIEW

File: engine/security/trust/index.md [hunk 1]
```diff
@@ -97,13 +97,15 @@ The following image depicts the various signing keys and their relationships:
...
-> tag from this repository prior to the loss.
+>The root key once lost is not recoverable. If you lose any other key, send an email to [Docker Hub Support](mailto:hub-support@docker.com). This loss also requires manual intervention from every
+consumer that used a signed tag from this repository prior to the loss.
 {:.warning}
 
+
+
+
+
+
```
Comment (dvdksn) [inline engine/security/trust/index.md:108]: There are a lot of empty lines here, it's best if we only have two newlines per block. Some markdown renderers would render empty paragraphs here which would create a big margin.

File: engine/security/trust/trust_key_mng.md [hunk 2]
```diff
@@ -69,16 +71,18 @@ for more details.
 
 Prior to Docker Engine 1.11, this feature was only in the experimental branch.
 
-## Lost keys
+## Key Loss
```
Comment (dvdksn) [inline engine/security/trust/trust_key_mng.md:74]: nit

```suggestion
## Key loss
```

File: engine/security/trust/trust_key_mng.md [hunk 3]
```diff
@@ -69,16 +71,18 @@ for more details.
...
 
-If a publisher loses keys it means losing the ability to sign trusted content for
-your repositories.  If you lose a key, send an email to [Docker Hub
-Support](mailto:hub-support@docker.com) to reset the repository
-state.
+> **WARNING**
+>
+> If a publisher loses keys it means losing the ability to sign trusted content for the repositories in
```
Comment (dvdksn) [inline engine/security/trust/trust_key_mng.md:78]: I would probably steer clear of the term "trusted content" when we are talking about signed software. Use trusted content for DOI/DVP/DSOS.

```suggestion
> If a publisher loses keys it means losing the ability to sign images for the repositories in
```

**Review by dvdksn — CHANGES_REQUESTED** [review]
A couple minor comments, but the phrase "trusted content" I think we should be careful with in this context, going fwd

File: engine/security/trust/index.md (hunk 1 above)
Comment (dockertopia) [reply engine/security/trust/index.md:108]: I'm surprised the linter didn't get this. Isn't this being check there?

Comment (dockertopia) [reply engine/security/trust/index.md]: Addressed in commit 3.

File: engine/security/trust/trust_key_mng.md (hunk 2 above)
Comment (dockertopia) [reply engine/security/trust/trust_key_mng.md]: Addressed in commit 3.

File: engine/security/trust/trust_key_mng.md (hunk 3 above)
Comment (dockertopia) [reply engine/security/trust/trust_key_mng.md]: Addressed in commit 3.

**Comment by dockertopia** [comment]
@dvdksn as discussed looped you in as I thought this content would interest you. Thanks for your suggestions, can you PTAL.

**Review by dvdksn — APPROVED** [review]
🌟🌟🌟🌟🌟

**Pull request merged**